scripts/tls13.test

```diff
@@ -245,60 +245,74 @@ fi
 
 if [ "$early_data" = "yes" ]; then
     echo -e "\n\nTLS v1.3 Early Data - session ticket"
-    port=0
-    (./examples/server/server -v 4 -r -0 -R "$ready_file" -p $port 2>&1 | \
-            tee "$server_out_file") &
-    server_pid=$!
-    create_port
-    ./examples/client/client -v 4 -r -0 -p $port 2>&1 >"$client_out_file"
-    RESULT=$?
-    cat "$client_out_file"
-    remove_ready_file
-    grep 'Session Ticket' "$client_out_file"
-    session_ticket=$?
-    early_data_cnt=`grep 'Early Data' "$server_out_file" | wc -l`
-    if [ $session_ticket -eq 0 -a $early_data_cnt -ne 4 ]; then
-        RESULT=1
-    fi
-    if [ $RESULT -ne 0 ]; then
-        echo -e "\n\nIssue with TLS v1.3 Early Data - session ticket"
+    counter=0
+    while [ "$counter" -lt 3 ]; do
```

Comment:
Don't need to test it multiple times.
I just remember having issue with greping the full string.

Reply:
Reverted.